### PR TITLE
[670] replace use of deprecated utcfromtimestamp

### PR DESCRIPTION
**Pull request by trel**
not tested, just find/replaced.

@d-w-moore please run this through testing

### CODE REVIEW

**Comment by d-w-moore** [comment]
> not tested, just find/replaced.
> 
> @d-w-moore please run this through testing

I'm getting one failure:
```
======================================================================
ERROR: test_timestamp_access_386 (irods.test.meta_test.TestMeta)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/daniel/python-irodsclient/irods/test/meta_test.py", line 567, in test_timestamp_access_386
    self.assertLess(create, now)    #  Ensure timestamps are in proper order.
  File "/usr/lib/python3.8/unittest/case.py", line 1296, in assertLess
    if not a < b:
TypeError: can't compare offset-naive and offset-aware datetimes
```

**Comment by d-w-moore** [comment]
```
I think because the also-deprecated `now = datetime.utcnow()` should be replaced by:
now = datetime.now( timezone.utc )
```
which creates an offset-aware reference for "now"...
Will test later.

**Comment by trel** [comment]
right - both of those are deprecated in 3.12.

**Comment by d-w-moore** [comment]
Seems to work with the addition of cfe0c2077730d10e266f403565d093397f462c9f . @trel I've made a PR to your repo....

**Comment by alanking** [comment]
We should consider cherry-picking that commit to this branch, or squash it into the existing commit in this PR with co-authorship attribution. Although, a PR for a PR is kind of a fun way to go about this... :)

**Comment by trel** [comment]
I'm going to absorb Dan's updates into this PR.

File: irods/test/query_test.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import unittest
 import time
 import uuid
-from datetime import datetime
+from datetime import (datetime, timezone)
```
Comment (trel) [inline irods/test/query_test.py:10]: why the parens here?  is that the new thing?

Comment (trel) [reply irods/test/query_test.py:10]: oh, it's for long lines... 

https://stackoverflow.com/questions/29534027/why-import-statements-with-parentheses

**Review by korydraughn — COMMENTED** [review]
Looks good.

Anything else left to do for this PR?
Are the tests passing?

**Comment by trel** [comment]
tests are passing - just awaiting clarity on the parentheses above.

doesn't seem to matter, it's just style/consistency.

**Comment by trel** [comment]
squashed and co-authored.

File: irods/test/query_test.py (hunk 1 above)
Comment (d-w-moore) [reply irods/test/query_test.py]: Yes, strictly optional for short ones.

**Review by korydraughn — APPROVED** [review]
If the tests are passing, pound it.

**Comment by trel** [comment]
@d-w-moore please assert the tests pass once more with the squashed single commit here.

**Comment by d-w-moore** [comment]
> @d-w-moore please assert the tests pass once more with the squashed single commit here.

running them through now.

**Review by alanking — APPROVED** [review]
Nice. # it

**Comment by korydraughn** [comment]
Sounds like this is ready for pounds.

@d-w-moore You agree?

**Comment by trel** [comment]
will # tonight

**Pull request merged**